detox/specs/activityIndicator.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { ActivityIndicator, Pages } from "./elements";
+import { device, expect } from "detox";
+
+describe("Activity Indicator", () => {
+    beforeAll(async () => {
+        await Pages().openActivityIndicator();
+    });
+
+    it("should render the default indicator", async () => {
+        await expect(ActivityIndicator("activityIndicator1")).toBeVisible();
+    });
+
+    it("should render the custom indicator", async () => {
+        await expect(ActivityIndicator("activityIndicator2")).toBeVisible();
+    });
+
+    afterAll(async () => {
+        await device.reloadReactNative();
```

Comment:
Maybe put this in the afterall of the startup js file

Reply:
Removed